latest/index.bs

```diff
@@ -154,6 +154,108 @@ multiple levels of resolutions and optionally associated labels.
+            │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+            │
+            └── col_0         # Each column in the var table is a 1D zarr array. The rows can be chunked as the user desires.
+                ├── .zarray   # However, the var columns SHOULD be chunked in the same way as the columns in X.
+                │
+                └─ 0
+```
+
+Points {#points-layout}
+-----------------------
+The following describes the expected layout for point annotations in OME-NGFF. Point annotations are tables with some specified columns and layout.
+
+```
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images by image ID.
+│
+└── 123.zarr
+    └── points                # The points group is a container which holds a table that is compatible with a subset of AnnData.
+        │                     # The points group MAY be in the root of the zarr file.
+        ├── .zgroup           # The table group MAY be in an image or labels group.
+        ├── .zattrs              
+        │
+        ├── X                 # You MUST add an zarr array `X`.
+        │   │                 # `X` MUST contain the coordinates for the points.
+        │   │                 # `X` MAY contain data in addition to the coordinates.
+        │   │                 # `X` MAY be chunked as the user desires.
+        │   ├── .zarray
+        │   ├── 0.0
+        │   │   ...
+        │   └── n.m
+        │        
+        ├── obs               # You MAY add an obs group container. The obs group holds a table of annotations for the points in X.
+        │   │                 # The rows in obs MUST be index-matched to the rows in X.
+        │   │                 
+        │   ├── .zgroup
+        │   │                     
+        │   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+        │   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+        │   │                 # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+        │   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        │   │      
+        │   └── col_0         # Each column in the obs table is a 1D zarr array. The rows can be chunked as the user desires.
+        │       ├── .zarray   # However, the obs columns SHOULD be chunked in the same way as the rows in X.
+        │       │
+        │       └─ 0
+        └── var               # You MUST add a var group container. The var group holds a table of annotations on the columns in X.
+            |                 # The var group MUST contain a column called `column_name` that annotates what each column in X contains.
```

Comment:
@constantinpape , we should coordinate on how to name the axes for the coordinates in X.

Reply:
- If points provide coordinates for the dimension axes, `dimension_label`/`dimension_name` seems fitting to me (although if non-coordinates columns in X are allowed, column name would be more general). We maybe could also require to just use the column specified in `_index` (as in table spec) for this purpose.

- I would favor column names to match axes names of coordinates (t, c, z, y, x or others).

Reply:
For reference, the exact scheme for labeling axes is still discussed in #57. But I think there are no big objections to labeling it something like `{"axes": {"labels": ["x", "y", ...]}}` (maybe `"names"` instead of `"labels"`). So I think we should use the same names as indicated there.